src/dispatch/static/dispatch/src/components/PreciseDateTimePicker.vue

```diff
@@ -0,0 +1,334 @@
+              @click="highlightText"
+            />
+          </v-col>
+          <span class="slash-space">/</span>
+          <v-col cols="1" class="pa-0">
+            <v-text-field
+              v-model="day"
+              class="m-0"
+              variant="plain"
+              density="compact"
+              hide-details="true"
+              maxlength="2"
+              @update:model-value="filterNumeric('day', $event)"
+              @blur="validateDay"
+              @focus="highlightText"
+              @click="highlightText"
+            />
+          </v-col>
+          <span class="slash-space">/</span>
+          <v-col cols="2" class="pa-0">
+            <v-text-field
+              v-model="year"
+              class="m-0"
+              variant="plain"
+              density="compact"
+              hide-details="true"
+              maxlength="4"
+              @update:model-value="filterNumeric('year', $event, true)"
+              @blur="validateYear"
+              @focus="highlightText"
+              @click="highlightText"
+            />
+          </v-col>
+          <span class="slash-space">,&nbsp;</span>
+          <v-col cols="1" class="pa-0">
+            <v-text-field
+              v-model="hour"
+              class="m-0"
+              variant="plain"
+              density="compact"
+              hide-details="true"
+              maxlength="2"
+              @update:model-value="filterNumeric('year', $event)"
+              @blur="validateHour"
+              @focus="highlightText"
+              @click="highlightText"
+            />
+          </v-col>
+          <span class="colon-space">:</span>
+          <v-col cols="1" class="pa-0">
+            <v-text-field
+              v-model="minutes"
+              class="m-0"
+              variant="plain"
+              density="compact"
+              hide-details="true"
+              maxlength="2"
+              @update:model-value="filterNumeric('year', $event)"
+              @blur="validateMinutes"
+              @focus="highlightText"
+              @click="highlightText"
+            />
+          </v-col>
+          <span class="colon-space">:</span>
+          <v-col cols="3" class="pa-0">
+            <v-text-field
+              v-model="seconds"
+              class="m-0"
+              variant="plain"
+              density="compact"
+              hide-details="true"
+              maxlength="6"
+              @update:model-value="filterNumeric('year', $event)"
+              @blur="validateSeconds"
+              @focus="highlightText"
+              @click="highlightText"
+            />
+          </v-col>
+        </v-row>
+        <div class="text-caption font-weight-light">{{ timezone || "UTC" }}</div>
+      </v-container>
+      <div class="mt-3 mb-3">Or paste a Unix timestamp:</div>
+      <v-text-field
+        class="paste-only"
+        v-model="unixTimestamp"
+        label="Unix Timestamp (UTC)"
```

Comment:
This looks like it is a field for ISO Timestamp, not the unix timestamp. If it is for ISO we should not only accept and parse `Z` formats.